### PR TITLE
Add notice about JS bundle size

### PR DESCRIPTION
**Pull request by i-like-robots**
Just a little thing ¯\_(ツ)_/¯

screen shot 2016-01-06 at 13 36 37

screen shot 2016-01-06 at 13 48 50

### CODE REVIEW

**Comment by matthew-andrews** [comment]
:+1: :dollar:

File: tasks/build.js [hunk 1]
```diff
@@ -82,6 +83,12 @@ function buildJs(jsFile) {
 	})
 	.on('end', function() {
 		console.log('build-js completed for ' + jsFile);
+
+		const bytes = fs.statSync(`${buildFolder}/${jsFile}`).size;
+		// EE roaming charge, Jan 2016
+		const costPerMb = 1.25
```
Comment (matthew-andrews) [inline tasks/build.js]: Three mobile is £6 per megabyte :cry:

**Comment by matthew-andrews** [comment]
Only for JS not SCSS?

**Comment by i-like-robots** [comment]
Added it for CSS too =]

**Comment by wheresrhys** [comment]
:+1::+1:

**Comment by keirog** [comment]
:+1: yessss

**Comment by i-like-robots** [comment]
Good point @wheresrhys, that should be reflected in the price. The full size could also be used to estimate memory usage of the AST (worth considering on low-powered/older mobiles) but this was just a 10 min hack.

**Pull request merged**